lib/block-supports/settings.php

```diff
@@ -0,0 +1,119 @@
+<?php
+/**
+ * Block level presets support.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Get the class name used on block level presets.
+ *
+ * @access private
+ *
+ * @param array $block Block object.
+ * @return string      The unique class name.
+ */
+function _gutenberg_get_presets_class_name( $block ) {
```

Comment:
Could we use `wp_unique_id` instead of introducing a new function for creating the class?

Reply:
Nah, we can't, unless we make it static in a function, which is about the same we have here.

Reply:
Hi @oandregal,
We need the same class on two different hooks using wp_unique_id would return two different ids.
I'm using exactly the same approach used on elements:
```
function gutenberg_get_elements_class_name( $block ) {
	return 'wp-elements-' . md5( serialize( $block ) );
}
```